parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -85,15 +83,7 @@ private String getAuthenticationKey(Credential credential) {
 		final UserPasswordCredential userPasswordCredential = (UserPasswordCredential) credential;
 		final String username = userPasswordCredential.getUsername();
 		final String password = userPasswordCredential.getPassword();
-		final String data = username.toLowerCase() + "//" + password;
-		try {
-			final MessageDigest digest = MessageDigest.getInstance("SHA-256");
-			final byte[] hashBytes = digest.digest(data.getBytes());
-			return Base64.encodeBase64URLSafeString(hashBytes);
-		} catch (NoSuchAlgorithmException e) {
-			e.printStackTrace();
-		}
-		return null;
+		return DigestUtils.sha256Hex(username.toLowerCase() + "//" + password);
```

Comment:
Ah, the difference is that mine uses Base64 encoding instead of the (in this context) more widely-used hexadecimal encoding. The advantage of Base64 is that it saves space while still being URL-safe, but if LegionBoard uses hexadecimal, you obviously need to use that here too :wink: